frontend/src/metabase/containers/SaveQuestionModal/SaveQuestionModal.tsx

```diff
@@ -113,10 +113,21 @@ export const SaveQuestionModal = ({
     }
   }
 
+  const getOriginalNameModification = (originalQuestion: Question | null) =>
+    originalQuestion
+      ? originalQuestion?.displayName() + " - " + t`Modified`
```

Comment:
Redundant `?`

```suggestion
      ? originalQuestion.displayName() + " - " + t`Modified`
```

Reply:
Yep.
Thanks for catching. Leftover from before.